_data/projects/real-time-project.yml

```diff
@@ -0,0 +1,10 @@
+name: Real Time Communications World
+desc: Maintain API Security
+site: https://www.realtimecommunicationsworld.com
+tags:
+  - api
+  - ddos
+  - json
+  upforgrabs:
```

Comment:
I think the indents on this line and the next two need to be removed, as our parser expects an array for `tags` and these lines are being parsed as part of it.